docs/user_guide.md

```diff
@@ -589,3 +590,10 @@ If you are a system admin, you can view the latest 10 records of garbage collect
 You can click on the 'details' link to view the related logs.
 ![browse project](img/gc_details.png)
 
+## Build history
+
+Have you ever found an image on Harbor and wondered what code it was built from? Or tried to locate the image for a source code repository? Build history make it easy to see the contents of a container image.

```

Comment:
I don't think we need an interrogative sentence here. Just directly describe what `build history` can do.